pkg/driver/camera/camera.go

```diff
@@ -1 +1,16 @@
+/*
+Package camera provides a video camera driver.
+
+Device Label Generation Rules
+
+On Linux, the device label will be in the format of:
+	/dev/video0;/dev/v4l/by-path/pci-0000:00:00.0-usb-0:0:0.0-video-index0
+If /dev/v4l/by-path/* is not available (for example in a docker container without
+bindings in /dev/v4l/by-path/), it will be:
+	/dev/video0;/dev/video0
```

Comment:
As `label := filepath.Base(device)` at https://github.com/pion/mediadevices/pull/311/files#diff-a943fe45da1d43feb94b8bf268a7bd7a1a17964da06bf426aae7046c078a8b35R84, the document should be
```diff
 On Linux, the device label will be in the format of:
-  	/dev/video0;/dev/v4l/by-path/pci-0000:00:00.0-usb-0:0:0.0-video-index0
+ 	video0;/dev/v4l/by-path/pci-0000:00:00.0-usb-0:0:0.0-video-index0
  If /dev/v4l/by-path/* is not available (for example in a docker container without
bindings in /dev/v4l/by-path/), it will be:
-  	/dev/video0;/dev/video0
+ 	video0;/dev/video0
```

Reply:
```diff
- 	video0;pci-0000:00:00.0-usb-0:0:0.0-video-index0
+ 	pci-0000:00:00.0-usb-0:0:0.0-video-index0;video0
```